bin/rollem.js

```diff
@@ -6,7 +6,7 @@ const fs = require('fs')
 const join = require('path').join
 const argv = require('minimist')(process.argv.slice(2))
 
-const configFilename = argv.c || 'rollem.config.js'
```

Comment:
This is an unexpected behavior for rollem, the c should not be valid without a followup argument specifying the filename. The code expects argv.c to be either a string, or undefined.